TDAT: public void test get employee ( ) throws exception {  <NL> system . out . println ( " get employee " ) ;  <NL>  <NL> transaction bo instance = new transaction bo ( ) ;  <NL>  <NL> employee bo exp result = null ;  <NL> employee bo result = instance . get employee ( ) ;  <NL> assert equals ( exp result , result ) ;  <NL>  <NL>  / / todo review the generated test code and remove the default call to fail .  <NL> fail ( " the test case is a prototype . " ) ;  <NL>  }  COM: <s> test of get employee method of class edu </s>